README.md

```diff
@@ -311,6 +312,8 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 [peoplepickeronadaptivecard#js]:samples/bot-people-picker-adaptive-card/nodejs
 [peoplepickeronadaptivecard#csharp]:samples/bot-people-picker-adaptive-card/csharp
 [graph-activity-feed-broadcast#js]:samples/graph-activity-feed-broadcast/nodejs
+[identity-link-with-sso#nodejs]:samples/identity-link-with-sso/nodejs
```

Comment:
Let's change the folder name as per the capability, here we can use app- as we are using multiple capabilities.